src/main/java/decodes/tsdb/algo/RelativeHumidity.java

```diff
@@ -0,0 +1,108 @@
+package decodes.tsdb.algo;
+
+import decodes.tsdb.DbCompException;
+import decodes.tsdb.IntervalCodes;
+import ilex.var.NamedVariable;
+import opendcs.opentsdb.Interval;
+import org.opendcs.annotations.algorithm.Algorithm;
+import org.opendcs.annotations.algorithm.Input;
+import org.opendcs.annotations.algorithm.Output;
+
+@Algorithm(
+		description ="Takes two inputs, Air temperature and dew point, to calculate relative humidity" )
+public class RelativeHumidity
+	extends AW_AlgorithmBase
+{
+
+	@Input
+	public double temperature;
+	@Input
+	public double dewPoint;
+
+	@Output
+	public NamedVariable output = new NamedVariable("output", 0);
+
+
+	// Allow javac to generate a no-args constructor.
+
+	/**
+	 * Algorithm-specific initialization provided by the subclass.
+	 */
+	protected void initAWAlgorithm( )
+		throws DbCompException
```

Comment:
Put all the throws on the same line after the method.

Reply:
I was use copying the original style but I made all recommended changes, thank you for reviewing it!